<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>qtcsdr</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QMySpectrumWidget" name="widgetFFT" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QTabWidget" name="tabWidgetControls">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>243</y>
         <width>622</width>
         <height>220</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabWidgetControlsPage1">
        <attribute name="title">
         <string>Main</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Freq:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleAM">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>10</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>AM</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Offset:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleWFM">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>WFM</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleUSB">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>USB</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleLSB">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>10</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>LSB</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleNFM">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>NFM</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>VFO:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleCW">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>10</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>CW</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleTransmit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>10</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>TX</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleRun">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>10</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>RX</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="FP10">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>80</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>+10</string>
         </property>
        </widget>
        <widget class="QPushButton" name="FP1">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>80</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>+1</string>
         </property>
        </widget>
        <widget class="QPushButton" name="FP100">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>80</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>+100</string>
         </property>
        </widget>
        <widget class="QPushButton" name="FM100">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>130</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-100</string>
         </property>
        </widget>
        <widget class="QPushButton" name="FM10">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>130</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-10</string>
         </property>
        </widget>
        <widget class="QPushButton" name="FM1">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>130</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-1</string>
         </property>
        </widget>
        <widget class="QPushButton" name="VFOG">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>80</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true">000</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="VFOM">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>80</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true">028</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="VFOK">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>80</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true">000</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="VFOH">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>80</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true">000</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="FQM">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>110</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true">028</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="FQK">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>110</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true">200</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="FQG">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>110</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true">000</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="FQH">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>110</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true">000</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="OFK">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>140</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true">200</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="OFH">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>140</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true">000</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="OFM">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>140</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabRTL">
        <attribute name="title">
         <string>RTL</string>
        </attribute>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Samp. rate:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboSampRate">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>2400000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2048000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1200000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1024000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>960000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>480000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>240000</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="comboDirectSamp">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>291</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Quadrature sampling</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Direct sampling (I branch)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Direct sampling (Q branch)</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>GAIN:</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="RTLSAGC">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Soft AGC</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="RTLHAGC">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>RTL AGC</string>
         </property>
        </widget>
        <widget class="QSlider" name="RTLGSlider">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>130</y>
           <width>471</width>
           <height>21</height>
          </rect>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSpinBox" name="RTLGTXT">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>130</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
        <widget class="QCheckBox" name="UPConv">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>10</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>UpConverter</string>
         </property>
        </widget>
        <widget class="QComboBox" name="UPConvVal">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>10</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>50</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>125</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>127</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tabCW">
        <attribute name="title">
         <string>CW</string>
        </attribute>
        <widget class="QSpinBox" name="CWwpmtxt">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="keyboardTracking">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>WPM:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Release time (ms):</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="CWrttxt">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>50</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="keyboardTracking">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QMySpectrumWidget</class>
   <extends>QWidget</extends>
   <header>qmyspectrumwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
